upgrading/automated_upgrades.adoc

```diff
@@ -519,10 +518,10 @@ on what additional features you have previously enabled.
 |Feature |Next Step
 
 |Aggregated Logging
-|xref:automated-upgrading-efk-logging-stack[Upgrade the EFK logging stack.]
+|xref:manual_upgrades.adoc#manual-upgrading-efk-logging-stack[Upgrade the EFK logging stack.]
 
 | Cluster Metrics
-| xref:automated-upgrading-cluster-metrics[Upgrade cluster metrics.]
+xref:../../manual_upgrades.adoc#manual-upgrading-cluster-metrics[Upgrade cluster metrics.]
```

Comment:
Cluster Metrics

Should show Aggregated Logging and Cluster Metrics  in the table, but Cluster Metrics is missing in this PR
******************************************************************************************************************
Feature                      Next Step    
Aggregated Logging | Upgrade the EFK logging stack.
Cluster Metrics         | Upgrade cluster metrics.
******************************************************************************************************************